catalog/docs/src/main/resources/ManContents.adoc

```diff
@@ -182,3 +182,39 @@ No configuration is required.
 (IMG)
 
 . Click delete.
+
+
+== Metacard Validation Plugin
+
+The metacard validation plugin (`catalog-plugin-metacard-validation`) is a feature that allows the validation/invalidation of metacards when they are ingested.
+Without this plugin, if validation is enabled, metacards are either valid and allowed into the catalog or are invalid and disallowed into the catalog. Without this plugin, if validation is disabled,
+all metacards will be valid and allowed into the catalog. With this plugin, validation can be enabled, and then metacards will be marked by this plugin as valid or invalid.
+
+=== Enabling feature
+
+This feature is turned off by default and can be enabled by typing `features:install catalog-plugin-metacard-validation` in the command line while DDF is running.
```

Comment:
@ricklarsen do we have generic docs on how to install a feature? Wondering if we sometimes describe it from the Admin Console and other times from the command line.